### PR TITLE
fix(datacenter.drp): fix order ip modal from zerto

### PR DESCRIPTION
**Pull request by JeremyDec**
MANAGER-2473

### CODE REVIEW

File: client/app/dedicatedCloud/datacenter/drp/steps/ovh/mainPcc/index.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ angular
         },
       })
       .state('app.dedicatedClouds.datacenter.drp.ovh.mainPccStep.legacyOrderIp', {
-        controller: 'IpOrderCtrl',
-        templateUrl: 'ip/ip/order/ip-ip-order.html',
+        controller: 'IpLegacyOrderCtrl',
+        templateUrl: 'ip/ip/legacyOrder/ip-ip-legacyOrder.html',
```
Comment (marie-j) [inline client/app/dedicatedCloud/datacenter/drp/steps/ovh/mainPcc/index.js]: Could those be imported instead ? :)

File: client/app/ip/ip/legacyOrder/ip-ip-legacyOrder.controller.js [hunk 2]
```diff
@@ -55,6 +57,16 @@ angular
       this.$scope.backToContracts = () => this.backToContracts();
       this.$scope.getResumePrice = price => this.getResumePrice(price);
       this.$scope.confirmOrder = () => this.confirmOrder();
+
+      if (this.isOrderingFromDrp()) {
```
Comment (FredericEspiau) [inline client/app/ip/ip/legacyOrder/ip-ip-legacyOrder.controller.js]:
```js
  this.$scope.closeModal = () => this.closeModal();
} // $onInit

closeModal(){
  if (this.isOrderingFromDrp()) {
    return this.$state.go('^');
  } 

  this.$scope.resetAction();
}
```

File: client/app/dedicatedCloud/datacenter/drp/steps/ovh/mainPcc/index.js (hunk 1 above)
Comment (JeremyDec) [reply client/app/dedicatedCloud/datacenter/drp/steps/ovh/mainPcc/index.js]: As seen together, importing the template would result in a `../../../../../ip/ip ...` so let's keep it like this

File: client/app/ip/ip/legacyOrder/ip-ip-legacyOrder.controller.js (hunk 2 above)
Comment (JeremyDec) [reply client/app/ip/ip/legacyOrder/ip-ip-legacyOrder.controller.js]: As seen together, I'll keep it like this

**Pull request merged**